### PR TITLE
Test websockets

### PR DESCRIPTION
**Pull request by Michal-Suhak**

### CODE REVIEW

File: src/quokka_editor_back/routers/websockets.py [hunk 1]
```diff
@@ -43,7 +44,20 @@ async def forward_from_redis_to_websocket(
                 )
     finally:
         await pubsub.unsubscribe(channel_name)
-        await redis_client.close()
+
+
+async def forward_from_redis_to_websocket(
+    websocket: WebSocket, document_id: UUID, user_token: str
+):
+    redis_client = await get_redis()
```
Comment (DamianCzajkowski) [inline src/quokka_editor_back/routers/websockets.py:52]: In the future maybe we can think about context-manager

File: src/quokka_editor_back/tests/routers/test_websockets.py [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+    message = {"data": b'{"key": "value"}'}
+
+    # When
+    result = decode_redis_message(message)
+
+    # Then
+    assert result == {"key": "value"}
+
+
+# TODO add test for subscribe_channel_and_broadcast_redis_messages
```
Comment (DamianCzajkowski) [inline src/quokka_editor_back/tests/routers/test_websockets.py:30]: add this todo to subscribe_channel_and_broadcast_redis_messages function

File: src/quokka_editor_back/tests/routers/test_websockets.py [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+    AsyncMock(spec=WebSocket)
+    mocked_manager_broadcast = mocker.patch(
+        "quokka_editor_back.routers.websockets.manager.broadcast", AsyncMock()
+    )
+    token = auth_handler.encode_token(active_user.username)
+
+    # When
+    with client.websocket_connect(f"/ws/{document.id}?token={token}") as websocket:
+        # Then
+        # mock_websocket.receive_text.assert_called_once_with()
```
Comment (DamianCzajkowski) [inline src/quokka_editor_back/tests/routers/test_websockets.py:289]: we are not checking it? if not remove this comment

**Review by DamianCzajkowski — APPROVED** [review]

File: src/quokka_editor_back/tests/routers/test_websockets.py (hunk 3 above)
Comment (Michal-Suhak) [reply src/quokka_editor_back/tests/routers/test_websockets.py:289]: These tests for websocket don't work

**Pull request merged**